### PR TITLE
Moved location and flushed out mechanics of dialogue, along with other changes

### PR DESCRIPTION
**Pull request by Dweblenod**
In addition to what is written in the commit message, I did some more reformatting(including of already existing files that hadnt been formatted) and changed a ColorHandler function name to be slightly less ambiguous

### CODE REVIEW

**Comment by Dweblenod** [comment]
I may be one of the leading programmers active on the project so far but that does not mean I have anything except informal experience! Im sure there are some inappropriate usages of code that I am missing(like something not being static or private where it should be, etc)

File: src/main/java/com/mraof/minestuck/MinestuckConfig.java [hunk 1]
```diff
@@ -51,6 +52,8 @@ private Client(Builder builder)
 					.define("loginColorSelector", true);
 			echeladderAnimation = builder.comment("Allows control of standard speed for the echeladder rung \"animation\", or if it should have one in the first place.")
 					.defineEnum("echeladderAnimation", AnimationSpeed.NORMAL);
+			dialogueSpeed = builder.comment("Controls speed at which dialogue appears.")
```
Comment (amiwyn) [inline src/main/java/com/mraof/minestuck/MinestuckConfig.java:55]: nice addition!

File: src/main/java/com/mraof/minestuck/client/gui/DialogueScreen.java [hunk 2]
```diff
@@ -1,47 +1,64 @@
...
 	private static final int GUI_WIDTH = 256;
-	private static final int GUI_HEIGHT = 90;
+	private static final int GUI_HEIGHT = 180;
 	private static final int PORTRAIT_SIZE = 32;
 	
-	private String[] paragraphs;
-	private String[] options;
+	private static final int RESPONSE_TEXT_OFFSET = 40;
+	
+	private String[] dialogueText;
```
Comment (amiwyn) [inline src/main/java/com/mraof/minestuck/client/gui/DialogueScreen.java:23]: maybe add an 's' to make it clear its multiple "screens" of text shown

File: src/main/java/com/mraof/minestuck/client/util/MSKeyHandler.java [hunk 3]
```diff
@@ -120,7 +120,7 @@ public static void onKeyInput(InputEvent.KeyInputEvent event)	//This is only cal
 					"..."
 				};
 				
-				Minecraft.getInstance().displayGuiScreen(new DialogueScreen(paragraphs, test, options));
+				//Minecraft.getInstance().displayGuiScreen(new DialogueScreen(paragraphs, test, options, color));
```
Comment (amiwyn) [inline src/main/java/com/mraof/minestuck/client/util/MSKeyHandler.java:123]: you can remove that function entirely :D

**Review by amiwyn — CHANGES_REQUESTED** [review]
looking good! ill merge once you fix the conflicts

**Comment by amiwyn** [comment]
Hmm I fucked it up with git, I solved the conflicts but I lost your commit history. Im closing this, your changes are in my branch thanks!

**Comment by Dweblenod** [comment]
Sorry I hadnt gotten to make the changes in time, thank you for your work!

**Pull request closed without merging**